src/include/sof/lib/cache.h

```diff
@@ -23,7 +24,7 @@
 #define CACHE_INVALIDATE	1
 
 /* data cache line alignment */
-#if DCACHE_LINE_SIZE > 0
+#if DCACHE_LINE_SIZE > 0 && !CONFIG_BAYTRAIL
```

Comment:
We shouldn't have platform dependant settings in project-wide include files. Please move it to platform/cavs code.